doc/dev/developer_guide/tests-unit-tests.rst

```diff
@@ -66,7 +66,6 @@ compiled during the build process.
 
 Unit testing of CLI tools
 -------------------------
-
```

Comment:
@zdover23 Are you sure the reStructuredText will render correctly if you remove this newline?